src/lib/database/seed.ts

```diff
@@ -0,0 +1,75 @@
+import type { Database } from "./schema.ts"
+
+export const seed = async (db: Database) => {
```

Comment:
Should export and return a Seed interface so we can use this in the test?

Reply:
this is a good question and super controversial. I think people should hard-code seed parameters because it encourages understanding of the seed environment (which is long-lived) and the seed parameters never change, also fwiw this is only used in `createFake` right now which wouldn't really make the seed accessible

Reply:
i know it sounds insane to hard code an id because normally it is, but in practice the code is much more readable/predictable because it prevents multiple names for the same id

Reply:
here's an idea: I return a Seed object but it's a `Record<literal, literal>`, just so that there's typechecking? win win?

Reply:
https://github.com/seamapi/fake-seam-connect/pull/33/commits/ebc007ee5ea62b1b2dd015485a25d439c170bc94